pyorbital/orbital.py

```diff
@@ -917,7 +921,26 @@ def propagate(self, utc_time):
         return kep
 
 
+def _get_tz_unaware_utctime(utc_time):
+    """Return timzone unaware datetime object.
+
+    The input *utc_time* is either a timezone unaware object assumed to be in
+    UTC, or a timezone aware datetime object in UTC.
+    """
+    if not hasattr(utc_time, 'tzinfo') or utc_time.tzinfo is None:
+        return utc_time
+
+    if utc_time.tzinfo != pytz.utc:
+        raise AttributeError("UTC time expected! Parsing a timezone aware datetime object requires it to be UTC!")
+
+    return utc_time.replace(tzinfo=None)
```

Comment:
I think readability could be improved by separating the datetime path from the datetime64 path, for example 

```python
if isinstance(utc_time, dt.datetime):
    if utc_time.tzinfo and utc_time.tzinfo != pytz.utc:
        raise AttributeError
    return utc_time.replace(tzinfo=None)
return utc_time
```